### ISSUE TITLE
LuaSocket for Lua 5.1 on Windows

### ISSUE THREAD

**Issue by Starkkz** [pkulchenko/ZeroBraneStudio#431]
Hello, I tried the three available version of LuaSocket for Windows, it seems that none of them could've been loaded. The program I used to run it supports Lua 5.1.4, I've placed core.dll on socket/core.dll but I couldn't load it because apparently they only run under Lua 5.3 or Lua 5.2. I read from the features that ZeroBraneStudio supports Lua 5.1, could anyone please tell me which is the correct core.dll that I must run in order to get LuaSocket 3.0-rc1 (for Lua 5.1) on Windows 32bit application?

**Comment by pkulchenko** [comment]
@Starkkz, It's better to avoid opening tickets to ask a question; several options to get in touch are listed on the [community page](http://studio.zerobrane.com/community.html).

As to your issue, what exactly is the error you are getting with your application (before any modifications to ZBS)? Why are you trying to replace luasocket that comes with ZBS? It already comes with v3.0-rc1 (for Lua 5.1), so no modifications should be needed.

**Comment by Starkkz** [comment]
@pkulchenko: Sorry about that, I'm going to try to explain my problem as better as I can.

1.- I was trying to build LuaSocket 2.0.2 on Linux (and Lua 5.1.4) but I had no success.
2.- Looking through the internet, I found out that ZBS had all the LuaSocket libraries built (and the newest ones)
3.- I tried the LuaSocket 3.0-rc1 build on a Linux 32bit program (Lua 5.1.4), and it loads correctly.
4.- I didn't have the windows build for the 32bit version of the same program (Lua 5.1.4)
5.- I downloaded the Windows build from the ZBS git repository and tried to load it
6.- The program was randomly crashing and I just thought I could use the clib52 and clib53
7.- Both clibs didn't work, they aren't compatible.
8.- I ended up opening a issue ticket here.

And this is what it outputs, no actual Lua errors.
[img]http://i.imgur.com/UIst3zF.jpg[/img]
